src/servers/public-api/books/books.controller.ts

```diff
@@ -48,7 +49,16 @@ export class BooksController {
     @Next() next: ExpressNextFunction
   ) {
     try {
-      const book = await services.books.sortBook(request.body);
+      const page = serializePagination(request).page;
+      const limit = serializePagination(request).limit;
+      const startIndex = (page - 1) * limit;
+      const countData = await services.books.countBook(request.body.filter)
```

Comment:
Sir, I already create the count method. However, I don't know how to access it in serializePagination.util so I can use it to find total records.